<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>20</y>
      <width>174</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="actionsLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="loadStorageButton">
       <property name="text">
        <string>Load Storage</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadTimetableButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Load Timetable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editTimeTableButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Edit Timetable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="simulateButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Simulate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="statisticsButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Simulation Statistics</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="storageGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>391</width>
      <height>601</height>
     </rect>
    </property>
    <property name="title">
     <string>Storage</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>371</width>
       <height>561</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="simulationsResultsGroupBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>501</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>Simulation Results</string>
    </property>
    <widget class="QTabWidget" name="daysTabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>901</width>
       <height>561</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRadioProgramManager">
    <property name="title">
     <string>RadioProgramManager</string>
    </property>
   </widget>
   <addaction name="menuRadioProgramManager"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
